src/common/CustomIconButton/CustomIconButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import {
+  EuiButton,
+} from '@elastic/eui';
+
+const CustomIconButton = ({
+  iconType, href, target, label,
+}) => {
+  if (process.env.NODE_ENV === 'test') {
+    return (
+      <EuiButton
```

Comment:
Won't you need `{label}` here? In case you want to check an Icon label during testing.

Reply:
Added.